app/etc/config.php

```diff
@@ -255,6 +255,8 @@
         'Magento_WishlistGraphQl' => 1,
         'PayPal_Braintree' => 1,
         'PayPal_BraintreeGraphQl' => 1,
-        'Temando_ShippingRemover' => 1
+        'Temando_ShippingRemover' => 1,
+        'Training_Sales' => 0,
```

Comment:
what is it?

Reply:
What is it what?
You mean 'Training_Sales' => 0,
or 
       -  'Temando_ShippingRemover' => 1
      +  'Temando_ShippingRemover' => 1,
      ???